tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -40,20 +42,22 @@ bool IsRetriable(Status status) {
 Status CallWithRetries(const std::function<Status()>& f) {
```

Comment:
Should `initial_delay_seconds` be made a optional arg to `CallWithRetries` with a default of 1? (Rather than it being a member variable and optional constructor parameter)